iModelCore/ECPresentation/Tests/NonPublished/Integration/Content/Content_RelatedPropertiesTests.cpp

```diff
@@ -1452,16 +1573,133 @@ TEST_F(RulesDrivenECPresentationManagerContentTests, LoadsXToManyRelatedInstance
+            "Type": {
+                "ValueFormat": "Primitive",
+                "TypeName": "string"
+                }
+            },{
+            "Name": "%s",
+            "Label": "ArrayProperty",
+            "Type": {
+                "ValueFormat": "Array",
+                "TypeName": "int[]",
+                "MemberType": {
+                    "ValueFormat": "Primitive",
+                    "TypeName": "int"
+                    }
+                }
+            },{
+            "Name": "%s",
+            "Label": "StructProperty",
+            "Type": {
+                "ValueFormat": "Struct",
+                "TypeName": "MyStruct",
+                "Members": [{
+                    "Name": "IntProperty",
+                    "Label": "IntProperty",
+                    "Type": {
+                        "ValueFormat": "Primitive",
+                        "TypeName": "int"
+                        }
+                    },{
+                    "Name": "StringProperty",
+                    "Label": "StringProperty",
+                    "Type": {
+                        "ValueFormat": "Primitive",
+                        "TypeName": "string"
+                        }
+                    }]
+                }
+            },{
+            "Name": "%s",
+            "Label": "StructArrayProperty",
+            "Type": {
+                "ValueFormat": "Array",
+                "TypeName": "MyStruct[]",
+                "MemberType": {
+                    "ValueFormat": "Struct",
+                    "TypeName": "MyStruct",
+                    "Members": [{
+                        "Name": "IntProperty",
+                        "Label": "IntProperty",
+                        "Type": {
+                            "ValueFormat": "Primitive",
+                            "TypeName": "int"
+                            }
+                        },{
+                        "Name": "StringProperty",
+                        "Label": "StringProperty",
+                        "Type": {
+                            "ValueFormat": "Primitive",
+                            "TypeName": "string"
+                            }
+                        }]
+                    }
+                }
+            }]
+        })",
+        FIELD_NAME(childClass2, "Parent"), FIELD_NAME(childClass2, "IntProperty"), FIELD_NAME(childClass2, "StringProperty"),
+        FIELD_NAME(childClass2, "ArrayProperty"), FIELD_NAME(childClass2, "StructProperty"),
+        FIELD_NAME(childClass2, "StructArrayProperty")).c_str());
+    rapidjson::Document actualChildClass2FieldType = descriptor->GetVisibleFields()[2]->GetTypeDescription().AsJson();
+    EXPECT_EQ(expectedChildClass2FieldType, actualChildClass2FieldType)
+        << "Expected: \r\n" << BeRapidJsonUtilities::ToPrettyString(expectedChildClass2FieldType) << "\r\n"
+        << "Actual: \r\n" << BeRapidJsonUtilities::ToPrettyString(actualChildClass2FieldType);

```

Comment:
do these type assertions have any virtue for this test?

Reply:
Removed